### ISSUE TITLE
Wishlist: Links back to the MDN JavaScript documentation

### ISSUE THREAD

**Issue by isonno** [rse/es6-features#74]
It would be awesome if this had links (either on the left or on the example pages) to the relevant location in the [MDN JavaScript documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference).  That way if you wanted more details on a particular feature it'd be just one click a way.

I suggest the MDN JavaScript docs because I find them complete, well maintained and well written, but any similar on-line JavaScript language doc would work.

**Comment by rse** [comment]
Yes, either MDN and/or MDN links are a good idea. If I find time, I'll try to add those.
